src/com/untamedears/PrisonPearl/PrisonPearlCommands.java

```diff
@@ -604,6 +604,10 @@ private boolean imprisonCmd(CommandSender sender, String args[]) {
         	uuid = NameAPI.getUUID(args[0]);
         else
         	uuid = Bukkit.getOfflinePlayer(args[0]).getUniqueId();
+        if (uuid == null){
```

Comment:
I feel that it would cover the edge case if for whatever reason either uuid grabbing methods fail. If the .getuniqueid always returns a value, regardless of the players existence, off or online; is the value constant or random? I feel that would be best caught along with the null check, if constant.